calico-enterprise/network-policy/domain-based-policy.mdx

```diff
@@ -229,6 +229,13 @@ spec:
       destination:
         selector: color == 'red'
 ```
+## Staged DNS Policy
```

Comment:
As written, it sounds like we are covering up a bug making them change a flag for some inconsistency on our part. I suspect we just need to emphasize that aligning results in flow logs between the two types of policy is a choice. For example:

When you stage a DNS policy, the goal is to preview and test traffic flows to ensure the policy is correctly allowing or blocking before enforcement. When a DNS staged policy first tries to reach a specified domain address or networkset, there can be a delay in the host reaching the domain that can result in a “no match” for the domain in flow logs. This flow log condition does not occur in enforced policies, but if you want to align flow log behavior between staged and enforced policy, change the FelixConfiguration parameter from DNSPolicyMode to DelayDNSResponse.